en/gps_compass/gps_locosys_hawk_a1.md

```diff
@@ -0,0 +1,99 @@
+# LOCOSYS Hawk A1 GPS/GNSS
+
+The [LOCOSYS HAWK A1 GPS/GNSS receiver](https://www.locosystech.com/en/product/hawk-a1-LU23031-V2.html) is a dual frequency multi-constellation GNSS/GPS receiver compatible with PX4.
+
+Main features include:
+- Concurrent reception of L1 and L5 band signals
+- Supports GPS, GLONASS,BEIDOU, GALILEO, QZSS
+- Capable of SBAS (WAAS, EGNOS, MSAS, GAGAN)
+- Support 135-channel GNSS
+- Fast TTFF at low signal level
+- Free hybrid ephemeris prediction to achieve faster cold start
+- Default 5Hz, up to 10 Hz update rate (SBAS at 5Hz update rate only)
+- Build-in super capacitor to reserve system data for rapid satellite acquisition
+- Three LED indicator for Power, PPS and Data transmit 
+
+![Hawk A1](../../assets/hardware/gps/locosys_hawk_a1/locosys_hawk_a1_gps.png)
```

Comment:
FYI only used markdown style links where possible. Use HTML when you need to resize the image - and in that case it may be a hint you need to shrink the actual image.